zkevm-circuits/src/state_circuit/test.rs

```diff
@@ -77,7 +78,34 @@ fn test_state_circuit_ok(
 fn degree() {
     let mut meta = ConstraintSystem::<Fr>::default();
     StateCircuit::configure(&mut meta);
-    assert_eq!(meta.degree(), 18);
+    assert_eq!(meta.degree(), 19);
```

Comment:
curious: which constraint increased the degree? or just increase it now since the future rw_counter constaint of Pad  will increase it by one?